rts/lra/coordinator-war/src/main/webapp/WEB-INF/web.xml

```diff
@@ -0,0 +1,20 @@
+<web-app version="3.1" xmlns="http://xmlns.jcp.org/xml/ns/javaee"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://xmlns.jcp.org/xml/ns/javaee http://xmlns.jcp.org/xml/ns/javaee/web-app_3_1.xsd">
+    <!-- One of the way of activating REST Services is adding these lines, the server is responsible for adding the corresponding servlet automatically. If the src folder, org.jboss.as.quickstarts.rshelloworld.HelloWorld class has the Annotations to receive REST invocation-->
+    <display-name>Narayana LRA Coordinator</display-name>
+
+    <listener>
+        <listener-class>io.narayana.lra.coordinator.api.AppContextListener</listener-class>
+    </listener>
+
+    <servlet>
+        <servlet-name>LRACoordinator</servlet-name>
+        <servlet-class>io.narayana.lra.coordinator.api.Coordinator</servlet-class>
```

Comment:
it seems you don't consider the recovery coordinator (https://github.com/jbosstm/narayana/blob/master/rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/RecoveryCoordinator.java), do you?

Reply:
good catch, thanks :+1: I am gonna add it

Reply:
After a bit of digging, I realised that the recovery coordinator does not need to be declared as a servlet. `Coordinator` is already doing it for us ([here](https://github.com/jbosstm/narayana/blob/9e4cbbc67025a684230db46fb6b0de21f69773dc/rts/lra/coordinator/src/main/java/io/narayana/lra/coordinator/api/Coordinator.java#L149-L152))